### PR TITLE
Added grammar meanings to the words

### PR DESCRIPTION
**Pull request by muhdsalm**
This commit simply adds grammar meanings to the words that have grammar meanings, such as 'ke' also introducing subordinate clauses.

Also two more small things:
- Added 'out of' and 'made of' to the meaning of 'de'
- Added an example to 'a', 'Bob a man', to show that a can be used as 'is' for nouns.

### CODE REVIEW

File: src/data/base_mini_dictionary.csv [hunk 1]
```diff
@@ -47,7 +47,7 @@ asilu;asylum, refuge, shelter
 atira;attract, attraction
 ato;act, performance, behavior, behaviour, active
 atomo;atom, atomic
-ave;have, possession
+ave;have, possession, [grammar] perfect tense
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: I don't think a new syntax is needed. We can just use "marks perfect tense".

File: src/data/base_mini_dictionary.csv [hunk 2]
```diff
@@ -119,7 +119,7 @@ danja;danger, dangerous
 danke;thanks, to thank, thankful
 dansa;dance
 dare;dare, daring, bold, brave, to challenge
-de;from, of, about, by
+de;from, of, about, by, out of, made of, [grammer] past tense
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: "marks past tense"

File: src/data/base_mini_dictionary.csv [hunk 3]
```diff
@@ -179,7 +179,7 @@ eleki;electric, electricity
 eleti;elect, election, select, choose, selection
 elika;helix, curl, twist, curly, spiral, coil, propeller
 emo;emotion, emotional, passionate, emote
-en;in, at, on
+en;in, at, on, [grammar] progressive tense
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: "marks progressives tense".

File: src/data/base_mini_dictionary.csv [hunk 4]
```diff
@@ -265,7 +265,7 @@ gene;inherit, inheritance, genetic, gene
 gera;war
 giga;billion
 gitara;guitar, lute, a strummed instrument
-go;to, go, towards, for
+go;to, go, towards, for [grammer] future
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: "marks future tense"

File: src/data/base_mini_dictionary.csv [hunk 5]
```diff
@@ -360,7 +360,7 @@ kata;cut, incision, slice, chop
 katana;sword
 kavalo;horse, donkey, mule
 kave;cave, cavern
-ke;what, that
+ke;what, that, [grammer] introduce dependant clause
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: "introduces dependant clause"

File: src/data/base_mini_dictionary.csv [hunk 6]
```diff
@@ -430,7 +430,7 @@ lakoni;short (length), curt, brief
 lampa;lamp
 lan;land, territory, domain, state
 lana;wool
-lase;let, allow, license, permission, licit
+lase;let, allow, license, permission, licit, [grammar] hortative mood
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: "marks hortative mood"

File: src/data/base_mini_dictionary.csv [hunk 7]
```diff
@@ -635,7 +635,7 @@ parada;parade, march, procession
 paren;parent
 pari;bet, wager, gamble
 parija;grill, grid, grate, broiler, gridiron
-pasa;pass, happen, past
+pasa;pass, happen, past, [grammar] habitual tense
```
Comment (jProgr) [inline src/data/base_mini_dictionary.csv]: "marks habitual mood".

File: src/data/mini_dictionary.csv
Comment (jProgr) [inline src/data/mini_dictionary.csv:1]: Same observations as with `base_mini_dictionary.csv`.

File: src/data/mini_dictionary.csv [hunk 8]
```diff
@@ -227,8 +227,8 @@ atomo-118;oganesson
 atomo-radi;radioactive
 atomo;atom, atomic
 Ausa;Hausa (people, language)
-Australia;Australia
```
Comment (jProgr) [inline src/data/mini_dictionary.csv:231]: Why?

**Review by jProgr — CHANGES_REQUESTED** [review]
Thank you very much!

**Comment by muhdsalm** [comment]
I have fixed all the problems you pointed out. If there's any other problems, let me know!

**Comment by jProgr** [comment]
@muhdsalm Thank you very much.

Finally, to merge your changes I need you to answer this: Are you aware and agree that your changes will be on the public domain? https://github.com/jProgr/buku-name/blob/master/LICENSE.md

**Comment by muhdsalm** [comment]
I am aware and I agree.

**Comment by jProgr** [comment]
Thank you very much, your changes have been merged.

**Pull request merged**